documentation/tutorials/getting-started-with-ash-authentication-phoenix.md

```diff
@@ -1,19 +1,30 @@
 # Getting Started Ash Authentication Phoenix
 
-In this step-by-step tutorial we create a new empty `Example` Phoenix application which provides the functionality to register users with their email address and a password. Registered users can sign in and out. Afterwards we will add the functionality to reset the password.
+In this step-by-step tutorial we create a new empty `Example` Phoenix + Ash application which provides the functionality for authentication. For beginners it is the best to follow the tutorial in the given order. For more advanced users it is a good reference to pick and choose from.
+
+- [Green Field Phoenix Application](#green-field-phoenix-application) (1 minute)
```

Comment:
This table of contents is auto generated on AshHQ. If you want it to appear in hex and github but not AshHQ, you can wrap it in these comments:

<!--- ash-hq-hide-start --> <!--- -->

your stuff goes here.

<!--- ash-hq-hide-stop --> <!--- -->